function/src/main/java/com/bloxbean/cardano/client/function/TxOutputBuilder.java

```diff
@@ -23,40 +23,26 @@ default TxOutputBuilder and(TxOutputBuilder after) {
         };
     }
 
-    @Deprecated
+    /**
+     * buildInputs
+     * Build Transaction Inputs by a TxInputBuilder and create new output for change output
+     *
+     * @param after TxInputBuilder to work with.
+     * @return {@link TxBuilder}
+     */
     default TxBuilder buildInputs(TxInputBuilder after) {
-        return (context, transaction) -> {
-            List<TransactionInput> inputs = new ArrayList<>();
-            List<TransactionOutput> outputs = new ArrayList<>();
-
-            this.accept(context, outputs);
-            TxInputBuilder.Result inputBuilderResult = after.apply(context, outputs);
-
-            if (!inputs.isEmpty())
-                transaction.getBody().getInputs().addAll(inputs);
-            transaction.getBody().getInputs().addAll(inputBuilderResult.inputs);
-
-            if (!outputs.isEmpty())
-                transaction.getBody().getOutputs().addAll(outputs);
-            if (inputBuilderResult.changes != null && !inputBuilderResult.changes.isEmpty())
-                inputBuilderResult.changes.forEach(txOutput -> {
-                    TransactionOutput txOutputSameAddress = transaction.getBody().
-                            getOutputs()
-                            .stream()
-                            .filter(transactionOutput -> transactionOutput.getAddress().equals(txOutput.getAddress()))
-                            .findFirst().orElse(null);
-                    if (txOutputSameAddress != null) {
-                        txOutputSameAddress.setValue(txOutputSameAddress.getValue().plus(txOutput.getValue()));
-                    } else {
-                        transaction.getBody().getOutputs().add(txOutput);
-                    }
-                });
-
-            //Clear multiasset in the context
-            context.clearMintMultiAssets();
-        };
+        return buildInputs(after, false);
```

Comment:
it's a mistake here we should send it with true not false.

Reply:
@edridudi I have updated separateChangeOutput to mergeChangeOutput

https://github.com/bloxbean/cardano-client-lib/commit/99e4dd5dbcf6fb57a789b50349dafd5d8d86ce7b

Reply:
Ok cool